naughty/fedora-37/3899-invalid-time-value

```diff
@@ -0,0 +1 @@
+RuntimeError: RangeError: Invalid time value
```

Comment:
This is way too little context. This needs to include parts of the traceback that restrict it to the affected test, otherwise this could easily apply to unrelated ones.

Reply:
Fixed.